cranelift-codegen/src/binemit/relaxation.rs

```diff
@@ -122,7 +122,11 @@ pub fn relax_branches(func: &mut Function, isa: &dyn TargetIsa) -> CodegenResult
     let jumptables_size = offset - jumptables;
     let rodata = offset;
 
-    // TODO: Once we have constant pools we'll do some processing here to update offset.
+    // TODO we are unable to do the following because BTreeSet does not have a mutable iterator (changing a set item could impact the order so this is not allowed)
+    //    for constant in func.constants.mut_iter() {
+    //        constant.offset(offset);
+    //        offset += constant.len() as u32;
+    //    }
```

Comment:
From the comment on the left it would seem that we need to update the constant offsets here but this is impossible (see TODO above).